code/Render/Types.h

```diff
@@ -428,6 +428,36 @@ struct Viewport
 	}
 };
 
+/*! Render scissor rectangle. */
+struct Rectangle
+{
+	int32_t left = 0;
+	int32_t top = 0;
+	int32_t width = 0;
+	int32_t height = 0;
+	Rectangle() = default;
+	Rectangle(int32_t l, int32_t t, int32_t w, int32_t h)
+		: left(l)
+		, top(t)
+		, width(w)
+		, height(h)
+	{
+	}
+
+	bool operator==(const Rectangle& other) const
```

Comment:
Why these operators? Doesn't seem to be necessary and if necessary should be using "= default".

Reply:
I'm using it in the RmlUi implementation. I'm caching the last set scissor. When RmlUi wants me to set set the scissor, I check the last set scissor against the new one to avoid unnecessary calls to the vulkan backend.
I updated to use the default implementation, but if you would prefer to not have it here, then I can just do a more verbose check in the RmlUi integration.

Reply:
I’ll remove the equality operator, rebase, and squash the fix up commits. I’m not sure if I remember right, but we may lost the review comments after a force push